tomviz/Utilities.h

```diff
@@ -178,6 +178,9 @@ QString readInPythonScript(const QString& scriptName);
 // of the built-in tomviz python operator scripts.
 QString readInJSONDescription(const QString& scriptName);
 
+// Get the path for the Tomviz directory
+QString getApplicationPath();
```

Comment:
I think this name is a little confusing, I would prefer something along the lines of `userDataPath` to signify the data path. We usually avoid the use of get too, it is a Qt convention we follow. I am not sure we should combine getting a path with creating a directory, but I think we can live with it for simplicity. The Python scripts are looking in `.tomviz` and `tomviz` as far as I remember.